edkrepo/commands/combo_command.py

```diff
@@ -45,6 +45,7 @@ def run_command(self, args, config):
             all_combos.extend(manifest.archived_combinations)
         if manifest.general_config.current_combo not in combo_list:
             combo_list.append(manifest.general_config.current_combo)
+            all_combos.extend(manifest.current_combination)

```

Comment:
can we accomplish this via the method used to add the current_combo to the combo_list variable in line 47 or the current_combo property defined on line 459 of the parser instead?

Reply:
current_combo is a string, and all_combos needs a Combination tuple which only combinations and archived_combinations returns.
we could search through every archived_combinations for one with a matching name and extend with that.
```
all_combos.extend(c for c in manifest.archived_combinations if c.name == manifest.general_config.current_combo)
```